src/batVect.ml

```diff
@@ -1165,4 +1197,36 @@ struct
   let print ?(first="[|") ?(last="|]") ?(sep="; ") print_a out t =
     BatEnum.print ~first ~last ~sep print_a out (enum t)
 
+  module Labels =
+  struct
+    let init n ~f      = init n f
+    let concat ~v1 ~v2 = concat v1 v2
```

Comment:
Regarding leaving out functions, I've mostly included all but some like `insert` and `remove` were not included as the data structure is the last argument anyways and there doesn't seem much reason to use the labelled version (at least to me). Again, I'll put it in if you say so.